### DISCUSSION TITLE
Alerts through SeqAPI

### DISCUSSION THREAD

**Discussion by spatel2-lh** [datalust/seq-tickets#1343] [Help] [answered]
Hello,

I am trying to query alerts through the SeqAPI and when I try to retrieve the Alerts resource group from the connection, I receive this error:

![image](https://user-images.githubusercontent.com/80535653/136587197-c07fdadc-7044-488b-9182-10eeaaca96be.png)

Here's the piece of code:

![image](https://user-images.githubusercontent.com/80535653/136587251-4683ee6b-2be7-4862-9b57-e8e2ecd2f069.png)

The version of SeqAPI I have is the latest: Seq.Api (2021.3.3-dev-00167). Any of the previous stable versions doesn't have the Alerts resource group available which now I'm wondering if it's still in progress.

Any help is appreciated.

Thanks!

**Comment by nblumhardt** [comment, marked as answer]
Hi @spatel2-lh - thanks for dropping by.

The 2021.3 version of _Seq.Api_ matches the 2021.3 version of Seq, which has a new alerting implementation. This is what `SeqConnection.Alerts` is providing access to.

Alerts in earlier Seq (and thus _Seq.Api_) versions are attached to charts under `SeqConnection.Dashboards`.

If you're doing any nontrivial manipulation of alerts, I'd recommend upgrading your Seq server to 2021.3 preview, which is only a few weeks from release, to avoid wasting effort migrating in the future.

Hope this helps!
Nick

**Reply by spatel2-lh** [reply]
Thanks Nick!

I will be on the lookout for 2021.3 and use SeqConnection.Dashboards in the meantime.
